src/App.css

```diff
@@ -115,3 +124,7 @@ body {
   max-height: 500px;
   overflow: hidden;
 }
+
+.max-w-510 {
+  max-width: 510px;
```

Comment:
you should be able to do this with `em` sizing, this way it will adjust to font size